molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/controller/MappingServiceController.java

```diff
@@ -650,8 +739,21 @@ public String advancedMappingEditor(@RequestParam(required = true) String mappin
 		AttributeMetaData targetAttribute = targetEntityMetaData != null ? targetEntityMetaData
 				.getAttribute(mappingServiceRequest.getTargetAttributeName()) : null;
 		Repository sourceRepo = dataService.getRepository(mappingServiceRequest.getSourceEntityName());
-		List<Object> calculatedValues = algorithmService.applyAlgorithm(targetAttribute,
+
+		Iterable<AlgorithmEvaluation> algorithmEvaluations = algorithmService.applyAlgorithm(targetAttribute,
```

Comment:
Can (and will) currently throw an exception at times.
What's the desired interface, should it be caught in the service or here?